assembl/static2/js/app/pages/surveyAdmin.jsx

```diff
@@ -4,19 +4,45 @@ import { I18n } from 'react-redux-i18n';
 
 import SectionTitle from '../components/administration/sectionTitle';
 import SurveyAdminForm from '../components/administration/survey';
+import { PHASES_ADMIN_MENU } from '../constants';
+import { getAdminMenuSection } from '../utils/administration/menu';
 
 type Props = {
-  section: string
+  section: string,
+  thematicId: string
 };
 
-const SurveyAdmin = ({ section }: Props) => {
-  const currentStep = parseInt(section, 10);
-  const sectionTitleMsgId = `administration.survey.${currentStep - 1}`;
+const getSectionTitle = (section: string, thematicId: string): string | null => {
+  if (PHASES_ADMIN_MENU.survey.subMenu.configThematics && thematicId) {
+    return 'administration.survey.configThematic';
+  }
+  const menuItem = getAdminMenuSection(section, PHASES_ADMIN_MENU.survey.subMenu);
+  return (menuItem && menuItem.title) || null;
```

Comment:
This could return `boolean | null` at runtime, so it's not consistent with the type signature of the method.

Reply:
(menuItem && menuItem.title) || null

1) menuItem = {title: 'foo'} => return 'foo'
2) menuItem = null or undefined => return null || null => return null

====> string | null

Reply:
Huh, been python-ing too long. I read `(menuItem && menuItem.title)` as returning a boolean. Fair enough.